metis-indexing/src/main/java/eu/europeana/indexing/mongo/WebResourceMetaInfoUpdater.java

```diff
@@ -25,8 +26,9 @@
 /**
  * Field updater for instances of {@link WebResourceMetaInfoImpl}.
  */
-public class WebResourceMetaInfoUpdater
-    extends AbstractMongoObjectUpdater<WebResourceMetaInfoImpl, WebResourceInformation> {
+public class WebResourceMetaInfoUpdater extends
```

Comment:
Please check my comment on MongoObjectUpdater interface.
I think there is another approach which will avoid this extra layer of interface, which I think provides a cleaner approach..
What do you think?